src/labkey/security/constants.ts

```diff
@@ -34,6 +34,7 @@ export enum PermissionRoles {
     FolderAdmin = 'org.labkey.api.security.roles.FolderAdminRole',
     ProjectAdmin = 'org.labkey.api.security.roles.ProjectAdminRole',
     Reader = 'org.labkey.api.security.roles.ReaderRole',
+    WorkflowEditor = 'org.labkey.samplemanagement.security.roles.WorkflowEditorRole',
```

Comment:
looks like this is the first non-platform role that has been added to this package. For example, the StorageDesignerRoel and StorageDataEditorRole were not added here. are we sure we want to put this SM role here?

Reply:
Ah, good question. I had originally put this in the api package like the others, but then moved it since it's sample-management specific. It probably better belongs in the workflow package. I'll move it there.